entity-service/src/main/kotlin/com/egm/stellio/entity/repository/QueryUtils.kt

```diff
@@ -206,17 +206,17 @@ object QueryUtils {
                 MATCH (user)
                 WHERE user.id IN ${'$'}userAndGroupIds
                 WITH user
-                MATCH (user)-[]->()-[right$authTerm]->$matchEntityClause
-                RETURN {entity: entity, right:right} as entity
+                MATCH (user)-[]->()-[right:$authTerm]->$matchEntityClause
+                RETURN {entity: entity, right: right} as entity
                 UNION
                 MATCH $matchEntityClause-[:HAS_VALUE]->(prop:Property { name: '$AUTH_PROP_SAP' })
                 WHERE prop.value IN [
                     '${AuthContextModel.SpecificAccessPolicy.AUTH_WRITE.name}',
                     '${AuthContextModel.SpecificAccessPolicy.AUTH_READ.name}'
                 ]
-                RETURN {entity:entity, right:"rCanRead"} as entity
+                RETURN entity
```

Comment:
this will gonna cause problems. the two variables are named `entity` (here and in line 210) but they don't contain the same thing

Reply:
btw, I guess it will break on line 219?